bench-vortex/benches/tpch_benchmark.rs

```diff
@@ -2,34 +2,61 @@ use bench_vortex::tpch::dbgen::{DBGen, DBGenOptions};
 use bench_vortex::tpch::query::Q1;
 use bench_vortex::tpch::{load_datasets, Format};
 use criterion::{criterion_group, criterion_main, Criterion};
-use tokio::runtime::Runtime;
+use tokio::runtime::Builder;
 
 fn benchmark(c: &mut Criterion) {
-    let runtime = Runtime::new().unwrap();
+    let runtime = Builder::new_current_thread()
```

Comment:
i split these out because i thought it'd make profiles easier to read, but it didn't actually make any difference. Will revert

Reply:
the two runtimes thing that is, i'm going to leave the change to use current-thread executor